onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -957,6 +963,395 @@ Status TreeEnsembleCommonClassifier<InputType, ThresholdType, OutputType>::compu
   return Status::OK();
 }
 
+template <typename IOType, typename ThresholdType>
```

Comment:
Why create another class instead of updating the exiting one like you did in #21222?